payloads/management/v3.5/configuration-api/index.js

```diff
@@ -21,4 +21,5 @@ export { default as listWebhookNames } from "./responses/webhooks/listWebhookNam
 export { default as getWebhooksState } from "./responses/webhooks/getWebhooksState.json"
 
 export { default as getOrganizationID } from "./responses/other/getOrganizationID.json"
-export { default as getLicenseID } from "./responses/other/getLicenseID.json"
\ No newline at end of file
+export { default as getLicenseID } from "./responses/other/getLicenseID.json"
+export { default as getProductChannelActivity } from "./responses/other/getProductChannelActivity.json"
```

Comment:
#optional: Add a new line to avoid two-line diff when someone else adds a line here

Reply:
added new line